Source/MoreDotNet.Test/Extensions/Common/RandomExtensions/OneOfTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.RandomExtensions
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using MoreDotNet.Extensions.Common;
+
+    using Xunit;
+
+    public class OneOfTests
```

Comment:
Missing test cases:
- What happens if the Random is null?
- What happens if the input list is null?